vertx-pojo-mapper-common/src/docs/asciidoc/groovy/index.adoc

```diff
@@ -23,7 +23,7 @@ To use this project, add the following dependency to the _dependencies_ section
 <dependency>
   <groupId>de.braintags</groupId>
 <artifactId>vertx-pojo-mapper-common</artifactId>
-<version>1.4.0-SNAPSHOT</version>
+<version>1.3.2-SNAPSHOT</version>
```

Comment:
Shouldn't  this remain at 1.4.0-SNAPSHOT ? Also see all the other similar changes